invesalius/gui/dialogs.py

```diff
@@ -416,7 +416,22 @@ def ShowImportOtherFilesDialog(id_type, msg='Import NIFTi 1 file'):
     return filename
 
 
+class CoordinateSpaceTransformHook(wx.FileDialogCustomizeHook):
+    def __init__(self):
+        super().__init__()
+        self.convertFromWorld = None
+
+    def AddCustomControls(self, customizer):
+        self.checkbox = customizer.AddCheckBox(_("File is in world/scanner coordinates"))
```

Comment:
Text is blue, like a "link", and does not look clear in dark theme. Try to format as regular text.